media/css/base/footer-refresh.scss

```diff
@@ -0,0 +1,569 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+$color-primary-gray: #181818;
+$color-secondary-gray: #757575;
+$color-tertiary-gray: #828282;
+$color-zilla-green: #00d230;
+$moz24-space-xl: 64px;
+$moz24-space-lg: 24px;
+$moz24-space-gutter: 32px;
+$moz24-space-grid-margin: 36px;
+$moz24-font-size-sm: 12px;
+$moz24-font-size-md: 16px;
+
+@font-face {
+    font-family: "Mozilla Sans Normal BETA05";
+    font-weight: 400;
+    font-style: normal;
+    font-display: swap;
+    src: local('Mozilla Sans Beta Regular');
+}
+
+@font-face {
+    font-family: "Mozilla Sans Normal BETA05";
+    font-weight: 600;
+    font-style: normal;
+    font-display: swap;
+    src: local('Mozilla Sans Beta SemiBold');
+}
+
+@font-face {
+    font-family: "Mozilla Semi-Slab Normal BETA05", "Zilla Slab", Inter, X-LocaleSpecific, sans-serif;
+    font-weight: 600;
+    font-style: normal;
+    font-display: swap;
+    src: local('Mozilla Sans Beta SemiBold');
+}
+
+.moz24-footer.mzp-c-footer {
+    padding-top: 0;
+}
+
+// newsletter form
+.moz24-newsletter-container {
+    background-color: $color-black;
+    color: $color-white;
+
+    &.c-newsletter.mzp-l-content {
+        padding-bottom: 0;
+    }
+
+    .moz24-newsletter-wrapper.c-newsletter-wrapper.mzp-l-content.mzp-t-content-lg {
+        padding: $moz24-space-xl 0 0;
+
+        @media(min-width: $screen-md) {
+            max-width: unset;
+            display: flex;
+            align-items: flex-start;
+        }
+    }
+
+    .moz24-newsletter-wrapper .c-newsletter-info {
+        display: flex;
+        align-items: flex-start;
+        margin-bottom: $moz24-space-grid-margin;
+
+        @media(min-width: $screen-md) {
+            width: 40%;
+            margin-right: 10%;
+        }
+
+        p {
+            margin-bottom: 0;
+            line-height: 1.3;
+            font-weight: 600;
+            font-family: "Mozilla Sans Normal BETA05", Inter, X-LocaleSpecific, sans-serif;
```

Comment:
**NIt:** it feels like this is repeated enough times for it to be a token/variable?